components/validator/guardian.go

```diff
@@ -102,7 +102,7 @@ func (g *Guardian) handleValidationRequested() {
 }
 
 func (g *Guardian) processOutputValidation(event *bindings.SecurityCouncilValidationRequested) {
-	ticker := time.NewTicker(time.Minute)
+	ticker := time.NewTicker(10 * time.Second)
```

Comment:
Why did you change?

Reply:
I thought it could be a little shorter since it's a duration value that applies when an output validation request comes in, not all the time.